src/helpers/CameraHelper.js

```diff
@@ -107,8 +107,8 @@ class CameraHelper extends LineSegments {
 		this.camera = camera;
 		if ( this.camera.updateProjectionMatrix ) this.camera.updateProjectionMatrix();
 
-		this.matrix = camera.matrixWorld;
-		this.matrixAutoUpdate = false;
+		this.matrixWorld = camera.matrixWorld;
+		this.matrixWorldAutoUpdate = false;
```

Comment:
This relied on the fact that world matrices were always updated even transiently. Since #28533, we can do this instead.